<launch>
  
<group>
  <machine name = "con_cen" address="192.168.0.3" user="sewoong" password="12341234" env-loader="/home/sewoong/catkin_ws/devel/env.sh" default="true">
  </machine>
  <node name="cmd_node" pkg="swarm_ctrl_pkg" type="cmd_node" output="screen">
    <param name="takeoff_alt" value="2"/>
  </node>

  <node name="state_node" pkg="swarm_ctrl_pkg" type="state_node" output="screen">
  </node>

  <node name="set_point_node" pkg="swarm_ctrl_pkg" type="set_point_node" output="screen">
   <param name="formation" value="diamond"/>
   <param name="offset" value="2.0"/>
 </node>

 <node name="control_node" pkg="swarm_ctrl_pkg" type="control_node" output="screen">
 </node>

</group>

</launch>
